src/main/scala/mesosphere/marathon/stream/RichTraversableLike.scala

```diff
@@ -0,0 +1,23 @@
+package mesosphere.marathon.stream
```

Comment:
Should all of our packages use the double-package notation? If not, what are the exceptions?